spec/support/helpers/download_helper.rb

```diff
@@ -0,0 +1,42 @@
+module DownloadHelpers
```

Comment:
I have some questions about the public API of this module. I understand what you are trying to achieve but it makes the test feel very magical. I think it would be good to pair on this for a bit.